### PR TITLE
Add support for parsing detached signatures

### PR DESCRIPTION
**Pull request by wiktor-k**
Note that I have *not* tested reading from bytes so I'll leave that as an exercise for @ktdreyer :)

Fixes: #27

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -427,11 +427,26 @@ private_parts = Cert.from_bytes(f"{c.secrets}".encode("utf8"))
...
+assert sig.issuer_fpr == "e8f23996f23218640cb44cbe75cf5ac418b8e74c"
+assert sig.created == datetime.fromisoformat("2023-07-19T18:14:01+00:00")
+```
+
 ## OpenPGP Cards
 
 There's an experimental feature allowing communication with OpenPGP
 Cards (like YubiKey or Nitrokey).
-
+ w
```
Comment (hko-s) [inline README.md:449]: :eyes:

**Comment by ktdreyer** [comment]
Thank you! That was fast.

- `from_bytes()` works perfectly in my testing with some RPM files.
- `signature.issuer_fpr` displays the full fingerprint string (40 hex characters, compared to PGPy's 16 characters)
- `signature.created` is a `datetime` object with the expected time.
- The README docs are clear.

**Comment by ktdreyer** [comment]
You can add "Fixes: #27" as a comment in this PR to have GitHub auto-close it.

**Comment by wiktor-k** [comment]
Okay, I've released 0.1.24 (both on PyPI and pushed a tag here). I guess that's it. Have a nice day! :wave:

**Pull request merged**